cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,32 @@
+class Iframe{
+    private URL: string;
+    private iFrameSelector: string;
+    private iFrameCssButtonSelector: string;
+
+    constructor(){
+        this.URL = "https://www.w3schools.com/html/html_iframe.asp"
+        this.iFrameSelector = "#main h1"
```

Comment:
I think that this selector is not for the iframe but for the title, so, it should be named as `frameTitle` or something similar